section1/reflection.md

```diff
@@ -2,18 +2,45 @@
+
 3. How would you log the string `"Hello World!"` to the console?
 
+`console.log("Hello World!");`
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+The characters used to designate comments are either two backslashes // for a single line following it or putting your comment inside of a backslash then asterisk for multi line code such as /* comment */. Code comments allow you to add more context about the code and explain what a certain section is doing. It can help both you and others understand your code easier. Especially helpful in a workplace where multiple people are working on the same code.
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable is a place to store a thing (string, number, boolean) and a way to refer to it later (variable name). You have to create a descriptive enough name to able to describe it accurately without it getting lost in a sea of bland variables. You can change the contents of the variable at a later time also. I would describe it to a 5 year old as a toybox. Like you can put different toys in the box (name = BuzzLightyear) but then you can replace him with (name = Woody) if you want to get rid of Buzz. But the toys have to have a name before you put them in there.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+Frontier Airlines. Three variables would be "numberOfTravelers" (integer), "promoCode" (string), and "arrivalAirport" (string).
+
 7. In your own words, explain what concatenation is.
 
+Concatenation means joining together variables and strings using the + symbol. Any number variable types will be converted to strings as well. It is essentially addition for different strings as opposed to numbers.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+Starbucks App. It has a welcome statement that says "Good afternoon, yourName" that has my name (Nicole) in it. So it is concatenating the existing generic welcome sentence with the variable of the person's name who's account it is, which will be different for each user.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+  * How do you decide how much detail to put in variable names without getting too verbose? like would "teacher" work or would "teacherName" be better?
+
+  * When would you use concatenation vs interpolation since they can produce the same result when printed to console?
+
+  * What is the github workflow when I am editing multiple documents at once? Do I commit each .js under exercises directory separately with appropriate commit messages or combine the commit messages into one?
+
+  * Can you see how your markdown formatting is working/ looking without committing the file first?
```

Comment:
Atom has a package that will let you preview! It's called `markdown-preview` - you can install in Preferences -> Packages, then use `ctrl + shift + m` when in a Markdown file - it will open up another tab in Atom with the preview